networking/networking.tex

```diff
@@ -78,7 +78,7 @@ \section{Layer 3: The Internet Protocol}
 One such in IPv4 is \keyword{127.0.0.1}, IPv6 as \keyword{0:0:0:0:0:0:0:1} or \keyword{::1} also known as localhost.
 Packets sent to 127.0.0.1 will never leave the machine; the address is specified to be the same machine.
 There are a lot of others that are denoted by certain octets being zeros or 255, the maximum value. You won't need to know all the terminology, keep in mind that the actual number of IP addresses that a machine can have globally over the Internet is smaller than the number of ``raw'' addresses.
-For the purposes of the class, you need to know at this layer that IP deals with routing, fragmenting, and reassembling upper-level protocols. A more in-depth aside follows.
+For the class, you need to know at this layer that IP deals with routing, fragmenting, and reassembling upper-level protocols. A more in-depth aside follows.
```

Comment:
Probably should just get rid of the `For the class` instead put in `This book covers`